skew/arn/__init__.py

```diff
@@ -192,14 +184,23 @@ class Region(ARNComponent):
                          'ap-northeast-1',
                          'sa-east-1']
 
+    _region_names_limited = ['us-east-1',
+                             'us-west-2',
+                             'eu-west-1',
+                             'ap-southeast-1',
+                             'ap-southeast-2',
+                             'ap-northeast-1']
+
+    _no_region_required = ['']
+
     _universal_region_names = ['us-east-1']
```

Comment:
Removed the _universal_region_names since it is no longer needed per your request.